### PR TITLE
Prepare rc8

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 - Read [this article](https://chris.beams.io/posts/git-commit/) before writing commit messages
 - Copy the file `commit-msg` to `.git/hooks`
+- Use `gradle build -x dokka` to build the source but exclude documentation jar generating to save time.
 - `gradle detektCheck` should not report any errors
 - This repo uses tabs! Make sure your code is properly formatted.
 - Use idea-code-style.xml for coding style .
 - We use [Spek](https://github.com/spekframework/spek) for testing.
 - Feel free to add your name to the contributors list at the end of the readme file when opening a pull request.
+- The code in `detekt-api`and any rule in `detekt-rules` must be documented. We generate documentation for our website based on this modules.
```
Comment (schalkms) [inline CONTRIBUTING.md]: missing space between `detekt-api` and *and*

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,11 +2,13 @@
 
 - Read [this article](https://chris.beams.io/posts/git-commit/) before writing commit messages
 - Copy the file `commit-msg` to `.git/hooks`
+- Use `gradle build -x dokka` to build the source but exclude documentation jar generating to save time.
```
Comment (schalkms) [inline CONTRIBUTING.md]: (y)

File: detekt-gradle-plugin/build.gradle.kts [hunk 3]
```diff
@@ -61,3 +71,91 @@ pluginBundle {
 		}
 	}
 }
+
+bintray {
+	user = System.getenv("BINTRAY_USER") ?: ""
```
Comment (schalkms) [inline detekt-gradle-plugin/build.gradle.kts]: I'm glad that you have automated the bintray publishing.

File: gradle.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
-detektVersion=1.0.0.RC7-3
+detektVersion=1.0.0.RC8
 usedDetektVersion=1.0.0.RC7-3
```
Comment (schalkms) [inline gradle.properties]: `usedDetektVersion=1.0.0.RC8`

Comment (arturbosch) [reply gradle.properties]: jap, after bintray stuff is uploaded to not get the CI errors :)

**Pull request merged**